core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/ReplicationStrategy.java

```diff
@@ -34,6 +34,8 @@ static ReplicationStrategy newInstance(Map<String, String> replicationConfig, St
         return new SimpleReplicationStrategy(replicationConfig);
       case "org.apache.cassandra.locator.NetworkTopologyStrategy":
         return new NetworkTopologyReplicationStrategy(replicationConfig, logPrefix);
+      case "org.apache.cassandra.locator.EverywhereStrategy":
+        return new EverywhereStrategy();
```

Comment:
All things considered, I think it would be cleaner to introduce an overridable factory. I can do it in a separate pull request if you want.

Reply:
:+1: to doing that in a separate PR,  just did this here to get things working in the near term.  Will back it out of my final commits after I finish testing everything.